acs-engine.go

```diff
@@ -122,6 +163,8 @@ var noPrettyPrint = flag.Bool("noPrettyPrint", false, "do not pretty print outpu
 var artifactsDir = flag.String("artifacts", "", "directory where artifacts will be written")
 var classicMode = flag.Bool("classicMode", false, "enable classic parameters and outputs")
 var parametersOnly = flag.Bool("parametersOnly", false, "only output the parameters")
+var caCertificatePath = flag.String("caCertificatePath", "", "the path to the CA Certificate file")
```

Comment:
Why do we plumb this into CLI but not other features in the apimodel.json? I guess maybe this is for convenience? Instead we could:

1. document it -- openssl and jq could inject them into the apimodel)
2. create a separate tool that would help spit out apimodel files based on flags (`acs-engine create-apimodel --orchestrator=k8s --ca-cert=blah.crt --ca-key=blah.key` which outputs the `apimodel.json`).

I worry a bit every apimodel knob is going to appear as a CLI flag if this starts a trend.

If you do leave this in, maybe make the flag a `StringPtr`. Then you can stop checking len = 0, and check against nil which seems more appropriate.